docs/source/tutorials/sklearn_svc.ipynb

```diff
@@ -0,0 +1,303 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        f = open("german.data-numeric")
</pre><p>where is the data file? I suggest that in the tutorial, we can obtain the data automatically via API or from the web</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tencent-quantum-lab/tensorcircuit/pull/164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
fixed!